src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const onlyWithSex =
+    person.sex === 'f'
+      ? `${person.partnerName} is my husband`
+      : `${person.partnerName} is my wife`;
+
+  return (
+    <section className="Person">
+      <>
+        <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+        {person.age && person.age !== 0 && (
+          <p className="Person__age">{`I am ${person.age}`}</p>
+        )}
```

Comment:
```suggestion
        {person.age && (
          <p className="Person__age">{`I am ${person.age}`}</p>
        )}
```
redundant to check on 0. if you want be sure not render 0, just make boolean !!person.age